app/src/main/java/com/varun/gbu_timetables/service/UpdateDatabaseService.java

```diff
@@ -45,27 +49,30 @@ public void onCreate() {
      */
     @Override
     public int onStartCommand(Intent intent, int flags, int startId) {
+
         if (updateDatabaseOnlineTask != null)
             if (updateDatabaseOnlineTask.getStatus() == AsyncTask.Status.RUNNING)
                 return mStartMode;
 
+
+
         updateDatabaseOnlineTask = new UpdateDatabaseOnlineTask(getApplicationContext(), true);
         updateDatabaseOnlineTask.execute();
 
         mStartMode = Service.START_NOT_STICKY;
 
-        /*NotificationCompat.Builder mBuilder =
+        NotificationCompat.Builder mBuilder =
                 new NotificationCompat.Builder(this)
                         .setSmallIcon(R.drawable.logo)
-                        .setContentTitle("My notification")
-                        .setContentText("Hello World!");
+                        .setContentTitle("Timetable Updated");
```

Comment:
This notification should come only when UpdateDatabaseOnlineTask updates the database and not each time when our service runs.